src/app/styles.ts

```diff
@@ -76,6 +76,19 @@ export const globalViewLight = css`
   a:hover > svg {
     fill: #007bff;
   }
+
+  .react-flow {
+    flex-grow: 1;
+    position: relative;
+    height: 100vh;
```

Comment:
This is not ideal. It should not be 100vh - it should just fill the remaining space and grow if needed.